goal_src/jak2/build/all_objs.json

```diff
@@ -0,0 +1,2252 @@
+["water-anim",                                 "water-anim",                 3, ["GAME", "COMMON"], "levels/common"],
+["blocking-plane",                             "blocking-plane",             3, ["GAME", "COMMON"], "levels/common"],
+["idle-control",                               "idle-control",               3, ["GAME", "COMMON"], "levels/common"],
+["dark-eco-pool",                              "dark-eco-pool",              3, ["GAME", "COMMON"], "levels/common"],
+["enemy-h",                                    "enemy-h",                    3, ["GAME", "COMMON"], "levels/common"],
+["nav-enemy-h",                                "nav-enemy-h",                3, ["GAME", "COMMON"], "engine/levels/nav"],
+["rigid-body-h",                               "rigid-body-h",               3, ["GAME", "COMMON"], "levels/common"],
+["enemy",                                      "enemy",                      3, ["GAME", "COMMON"], "levels/common"],
+["nav-enemy",                                  "nav-enemy",                  3, ["GAME", "COMMON"], "engine/levels/nav"],
+["base-plat",                                  "base-plat",                  3, ["GAME", "COMMON"], "levels/common"],
+["plat",                                       "plat",                       3, ["GAME", "COMMON"], "levels/common"],
+["basebutton",                                 "basebutton",                 3, ["GAME", "COMMON"], "levels/common"],
+["bouncer",                                    "bouncer",                    3, ["GAME", "COMMON"], "engine/levels/common/enemy"],
+["conveyor",                                   "conveyor",                   3, ["GAME", "COMMON"], "levels/common"],
+["elevator",                                   "elevator",                   3, ["GAME", "COMMON"], "levels/common"],
+["rigid-body",                                 "rigid-body",                 3, ["GAME", "COMMON"], "levels/common"],
+["rigid-body-queue",                           "rigid-body-queue",           3, ["GAME", "COMMON"], "levels/common"],
+["rigid-body-plat",                            "rigid-body-plat",            3, ["GAME", "COMMON"], "levels/common"],
+["joint-exploder",                             "joint-exploder",             3, ["GAME", "COMMON"], "levels/common"],
+["simple-focus",                               "simple-focus",               3, ["GAME", "COMMON"], "levels/common"],
+["simple-nav-sphere",                          "simple-nav-sphere",          3, ["GAME", "COMMON"], "levels/common"],
+["process-taskable",                           "process-taskable",           3, ["GAME", "COMMON"], "engine/game/task"],
+["scene-actor",                                "scene-actor",                3, ["GAME", "COMMON"], "levels/common"],
+["scene-looper",                               "scene-looper",               3, ["GAME", "COMMON"], "levels/common"],
+["warp-gate",                                  "warp-gate",                  3, ["GAME", "COMMON"], "levels/common"],
```

Comment:
from scene-actor on, levels/common seems good - these are pretty specific to jak 2 levels/art